documentation/docs/index.md

```diff
@@ -1,32 +1,45 @@
 # Project "Piper" User Documentation
 
-An efficient software development process is vital for success in building
-business applications on SAP Cloud Platform or SAP on-premise platforms. SAP
-addresses this need for efficiency with project "Piper". The goal of project
-"Piper" is to substantially ease setting up continuous delivery processes for
-the most important SAP technologies by means of Jenkins pipelines.
+Continuous delivery is a method to develop software with short feedback cycles.
+It is applicable to projects both for SAP Cloud Platform and SAP on-premise platforms.
+SAP implements tooling for continuous delivery in project "Piper".
+The goal of project "Piper" is to substantially ease setting up continuous delivery in your project using SAP technologies.
 
 ## What you get
 
-Project "Piper" consists of two parts:
+To get you started quickly, project "Piper" offers you the following artifacts:
 
-* [A shared library][piper-library] containing steps and utilities that are
-  required by Jenkins pipelines.
-* A set of [Docker images][devops-docker-images] used in the piper library to implement best practices.
+* A set of ready-made Continuous Delivery pipelines for direct use in your project
```

Comment:
```suggestion
* A set of preconfigured Continuous Delivery pipelines for direct use in your project
```

Reply:
"ready-made" is not a nice word, but "preconfigured" does not really describe what we want to convey.. What I want to say is that they are already coded, and you as the consumer don't have to code your own pipeline. Can you suggest a better wording for this, @SarahNoack?

Reply:
Pre-configured does not properly address the added value. So far use "ready-made" or "out-of-the-box". If that's something we should change, it needs to be addressed holistically.

Reply:
I would got with "ready-made" as it means "von der Stange" and that is from my understanding what we would like to express.